package.json

```diff
@@ -91,7 +92,6 @@
     "ember-cli-deploy-git": "^1.3.4",
     "ember-cli-htmlbars": "^5.3.2",
     "ember-cli-inject-live-reload": "^2.0.2",
-    "ember-cli-qunit": "^4.4.0",
```

Comment:
[`ember-cli-qunit`](https://github.com/ember-cli/ember-cli-qunit) has been deprecated. I replaced it with [`ember-qunit`](https://github.com/emberjs/ember-qunit).